### DISCUSSION TITLE
Clang Format 似乎没有正常运行

### DISCUSSION THREAD

**Discussion by ghost** [cpeditor/cpeditor-cn#36] [提问] [answered]
我已经将`clang-format`添加至系统环境变量`PATH`中，如图
![image](https://github.com/cpeditor/cpeditor-cn/assets/97910545/7d0fb6cc-706d-4865-b048-8d12e8426f10)
每当我格式化代码时就会爆红出错
![image](https://github.com/cpeditor/cpeditor-cn/assets/97910545/599ce1a5-4082-41cb-8885-b0f8e4f77119)

报错信息如下：
> [10:39:15] [Formatter] [The format command [clang-format --style=file --cursor=324 C:/Users/April/AppData/Local/Temp/cpeditor-qAYrTQ/tmp.cpp] finished with exit code 1.]
[10:39:15] [Formatter[stderr]] [
C:/Users/April/AppData/Local/Temp/cpeditor-qAYrTQ/.clang-format:1:1: error: not a mapping 
BasedOnStyle:Google 
^~~~~~~~~~~~~~~~~~~ 
Error reading C:/Users/April/AppData/Local/Temp/cpeditor-qAYrTQ/.clang-format: Invalid argument 
]

build info:
![image](https://github.com/cpeditor/cpeditor-cn/assets/97910545/acf30648-3849-4c4a-8626-824c20ee3693)

**Comment by ouuan** [comment, marked as answer]
clang format style 的冒号后面要有空格

**Reply by ghost** [reply]
问题解决了，谢谢
